src/SarifCli/AnalyzeOptions.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using CommandLine;
+using Microsoft.CodeAnalysis.Sarif.Driver;
+
+namespace Microsoft.CodeAnalysis.Sarif.Cli
+{
+    [Verb("validate", HelpText = "Validate a SARIF file against the schema and against additional correctness rules.")]
```

Comment:
>validate [](start = 11, length = 8)

you only have one arg, a path to a schema, why use a named arg? suggest a positional arg instead. also, your named args in general look exceeding verbose. this one could simply be --schema for example. imo